scripts/migrate-tags.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+source "$PWD/scripts/semver.sh"
+
+remote='origin'
+release_commits_regex='^[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}'
+version_before_package_rename='0.0.0'
+tag_prefix_before_package_rename="$1"
+dry_run=true
+
+while [[ $# -gt 0 ]]; do
+  key="$1"
+
+  case $key in
+  -r | --remote)
+    remote="$2"
+    shift # past argument
+    shift # past value
+    ;;
+  -p | --regex-pattern)
+    release_commits_regex="$2"
+    shift # past argument
+    shift # past value
+    ;;
+  -v | --version-before-package-rename)
+    version_before_package_rename="$2"
+    shift # past argument
+    shift # past value
+    ;;
+  -t | --tag-prefix-before-package-rename)
+    tag_prefix_before_package_rename="$2"
+    shift # past argument
+    shift # past value
+    ;;
+  --non-dry-run)
+    dry_run=false
+    shift # past argument
+    shift # past value
+    ;;
+  *) # package name
+    package_name="$1"
+    shift # past argument
+    ;;
+  esac
+done
+
+get-version-commit-pairs() {
+  while read -r log; do
+    echo "$log" | cut -d' ' -f1,2
+  done <<<"$(git log --oneline --grep $release_commits_regex merged-packages/$package_name)"
```

Comment:
The commit message might contain spaces, so I'm worried you might be cutting off part of the commit message if you split on space alone. We should also probably use full commit IDs rather than short versions (which `git log` returns by default).

I'm curious whether it would be less brittle if you asked `git log` to give you a formatted string where the commit ID and subject were separated by a tab character.

For instance, you could say:

``` suggestion
  git log --oneline --grep "$release_commits_regex" --format='%H%x09%s' "merged-packages/$package_name"
```